src/pyscaffold/log.py

```diff
@@ -196,6 +197,10 @@ def __init__(
         self.formatter = formatter or ReportFormatter()
         super(ReportLogger, self).__init__(self._wrapped, self.extra)
 
+        level = environ.get("PYSCAFFOLD_LOG_LEVEL")
```

Comment:
Should we document this environment variable `PYSCAFFOLD_LOG_LEVEL` somewhere or is it meant for development usage only?

Reply:
I used this to debug some errors that I was seeing and decided to leave as it is, supposing it could be useful in the future (specially when debugging things that happen before the CLI processes the verbosity level)...

I suppose final users should not have a direct usage for it... We could include it in the `log` module documentation but I would prefer not to commit on this being part of the public API (just an internal API, for PyScaffold developers only).

Reply:
Maybe we should revive this PYSCAFFOLD_LOG_LEVEL env var...